cmd/nbe/docker.go

```diff
@@ -160,6 +160,7 @@ func (r *ImageBuilder) Run() (string, error) {
 	c := exec.Command(
 		"docker",
 		"build",
+		// "--progress=plain", // Use plain output for debuggig
```

Comment:
1/5 leave it there, helps with debugging locally but not enough to make it an `nbe` flag